tils/dupal-8-dynamic-author-box.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: How to add a dynamic author box to a node (content type) in Drupal 8
+description: Quickly add a dynamic author box to any node type in Drupal 8 using an advanced Drupal view.
+author: Mudasir Ali
+date: '2021-07-16'
+category: Drupal
+---
+
```

Comment:
Can we reformat the list to use markdown format?

Reply:
Tried to reformat the list using markdown, the list bullets or numbers don't show up in the HTML.